apps/console/src/features/claims/components/add/add-external-claim.tsx

```diff
@@ -81,9 +83,9 @@ interface AddExternalClaimsPropsInterface extends TestableComponentInterface {
 /**
  * A component that lets you add an external claim.
  *
- * @param {AddExternalClaimsPropsInterface} props - Props injected to the component.
+ * @param props - Props injected to the component.
  *
- * @return {React.ReactElement} Component.
+ * @returns - Component.
```

Comment:
Please check and fix the other places as well.

```suggestion
 * @returns External claim add Component.
```